libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -0,0 +1,581 @@
+  color: var(--text-color);
+  white-space: no-wrap;
+}
+
+/* content hidden */
+.hero-marquee .hidden {
+  visibility: hidden;
+  height: 0;
+}
+
+.hero-marquee .foreground div:empty {
+  display: none;
+}
+
+/* Row Types */
+.hero-marquee .row-list {
+  text-align: left;
+} 
+
+html[dir="rtl"] .hero-marquee .row-list {
+  text-align: right;
+}
+
+.hero-marquee .row-list .row-wrapper {
+  display: table;
+}
+
+.hero-marquee .row-list li {
+  margin-bottom: var(--spacing-xxs);
+} 
+
+/* row-qr */
+.hero-marquee .row-qrcode .row-wrapper {
+  display: flex;
+  gap: var(--spacing-s);
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .row-wrapper > p {
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .qr-code-img {
+  display: none;
+}
+
+.hero-marquee .row-qrcode .google-play a,
+.hero-marquee .row-qrcode .app-store a {
+  width: 135px;
+  height: 40px;
+  display: inline-flex;
+  text-indent:-999em;
+}
+
+.hero-marquee .row-qrcode .google-play a {
+  background: url('/libs/img/ui/google-play.svg') no-repeat transparent;
+}
+
+.hero-marquee .row-qrcode .app-store a {
+  background: url('/libs/img/ui/app-store.svg') no-repeat transparent;
+}
+
+.hero-marquee.center .row-qrcode .row-wrapper {
+  justify-content: center;
+}
+
+.hero-marquee.center .row-list .row-wrapper { 
+  margin: 0 auto; 
+}
+
+.hero-marquee .row-list .icon-list {
+  padding-inline-start: var(--spacing-m);
+  margin: 0;
+}
+
+.hero-marquee li.icon-item {
+  position: relative;
+  list-style: none;
+}
+
+.hero-marquee li::marker {
+  padding-inline-start: 40px;
+}
+
+.hero-marquee li.icon-item::marker {
+  color: transparent;
+}
+
+.hero-marquee li.icon-item span.icon {
+  position: absolute;
+  left: -28px;
+}
+
+html[dir="rtl"] .hero-marquee li.icon-item span.icon {
+  left: unset;
+  right: -38px;
+}
+
+/* row-supplemental */
+.hero-marquee .row-supplemental.bold {
+  font-weight: 700;
+}
+
+.hero-marquee.fg-media-cover.has-bg .asset {
+  display: none;
+}
+
+.hero-marquee.asset-left > .foreground.cols-2 > .asset {
+  order: 2;
+}
+
+
+/* CLS */
+
+/* mobile ONLY */
+@media (max-width: 600px) {
+  .hero-marquee .con-button {
+    display: block;
+    text-align: center;
+  }
+  
+  .hero-marquee.fg-media-stack-top-mobile,
+  .hero-marquee.fg-media-stack-top-mobile-tablet {
+    flex-direction: column-reverse;
+  }
+
+  .hero-marquee.fg-media-stack-top-mobile .foreground .copy,
+  .hero-marquee.fg-media-stack-top-mobile-tablet .foreground .copy {
+    order: 2;
+  } 
+  
+  .hero-marquee.fg-media-cover:not(.bg-stack-bottom-mobile),
+  .hero-marquee.fg-media-cover:not(.fg-media-hidden-mobile) {
+    padding-bottom: 0;
+  }
+
+  /* con-vars support */
+  .hero-marquee.bg-stack-mobile .background,
+  .hero-marquee.bg-stack-top-mobile .background,
+  .hero-marquee.bg-stack-bottom-mobile .background,
+  .hero-marquee.bg-stack-mobile-tablet .background,
+  .hero-marquee.bg-stack-top-mobile-tablet .background,
+  .hero-marquee.bg-stack-bottom-mobile-tablet .background   {
```

Comment:
Maybe `:is` can shorten this one too

Reply:
True, Could also use a [class^="bg-stack-"]. I've updated to use :is and shortened it up a bit. There may be some other css selectors that could be optimized, ill keep an eye out.